app/gimletlet/src/main.rs

```diff
@@ -77,6 +77,17 @@ unsafe fn system_pre_init() {
         // spin
     }
 
+    // Turn on the internal RAMs.
+    let rcc = &*device::RCC::ptr();
+    rcc.ahb2enr.modify(|_, w| {
+        w.sram1en()
+            .set_bit()
+            .sram2en()
+            .set_bit()
+            .sram3en()
+            .set_bit()
+    });
+
```

Comment:
Want to add these changes to Sidecar's `main.rs` as well?

Reply:
Really, portions of this commit should be added to every stm32h7x3 main.rs, which serves as an excellent reminder that we need to centralize this stuff.

Reply:
I'd like to take centralizing this as a TODO after this merges, if it's okay with you.

Reply:
Sure, that makes sense – want to copypasta it into Sidecar in this PR, or should I do that in `task-vsc7448` after this is merged and I rebase?

Reply:
Go ahead and do the latter -- I've started the centralization work on a branch.